### PR TITLE
Replace JPigpio, add hardware PWM support

### PR DESCRIPTION
**Pull request by Bankst**

### CODE REVIEW

File: photon-server/src/main/java/org/photonvision/common/hardware/GPIO/pi/PigpioPin.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            }
+
+        } catch (PigpioException e) {
+            logger.error("Could not set blink - " + e.getMessage());
+        }
+    }
+
+    @Override
+    protected void setBrightnessImpl(int brightness) {
+        if (isHardwarePWMPin) {
```
Comment (pietroglyph) [inline photon-server/src/main/java/org/photonvision/common/hardware/GPIO/pi/PigpioPin.java:124]: Should we print an error if the current pin is not a hardware PWM pin?

Comment (Bankst) [reply photon-server/src/main/java/org/photonvision/common/hardware/GPIO/pi/PigpioPin.java:124]: Maybe once. I can also re-add softPWM and attempt using that, but I'd rather just stick to hardware PWM.

Comment (Bankst) [reply photon-server/src/main/java/org/photonvision/common/hardware/GPIO/pi/PigpioPin.java:125]: Resolved in 4cf21c8

**Review by XzibitGG — APPROVED** [review]

**Review by mcm001 — APPROVED** [review]

**Comment by Bankst** [comment]
Pending fix for LED slider resetting when changing in/out of drivermode. @gerth2 has reproduced this.

**Comment by mcm001** [comment]
The LED slider does not reset for me while running from this PR.

**Pull request merged**